tiatoolbox/models/architecture/unet.py

```diff
@@ -1,10 +1,10 @@
-"""Defines a set of ResNet variants to be used within tiatoolbox."""
+"""Defines a set of UNet variants to be used within tiatoolbox."""
 
 from typing import List, Tuple
 
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
+import torch.nn.functional as functional
```

Comment:
Not sure why this needed to be changed, `F` is the common form for pytorch code so it should be best to keep it in line with the community. You do no not need to edit but should keep this in mind.

Reply:
We discussed this in the last toolbox meeting. I agree that sticking to what the community does would make sense so that when people read our code, it matches up with other common code e.g.:

```python
import torch.nn.functional as F
from torch import nn
from matplotlib import pyplot as plt
# etc.
```